### ISSUE TITLE
Support for Python 3.2 +

### ISSUE THREAD

**Issue by 007bsd** [robotframework-thailand/robotframework-jsonlibrary#11]
**How to reproduce**
Not able to install the library for python 3.2 +. 

**Error**

> Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/fr/jtjjqpbs1sx0lxh6qbgzgdjw00gftq/T/pip-build-ghem6ueu/robotframework-jsonlibrary/

**Comment by MarkJB** [comment]
Assuming I'm seeing the same error (see below), adding the library directory to the import statement seems to fix it for me. I am then able to install the library to python 3. (Tested with Python 3.5.2 & 2.7 on linux).

If anyone else wants to try this workaround, download the library (from github or pypi) and edit the __init.py in the JSONLibrary directory so the imports on line 2 & 3 read as follow:

`from JSONLibrary.JSONLibraryKeywords import JSONLibraryKeywords`
`from JSONLibrary.version import VERSION`

and also edit JSONLibraryKeywords.py line 4:

`from JSONLibrary.version import VERSION`

I was then able to do a pip install of the locally modified library (just run pip install /path/to/local/robotframework-jsonlibrary/). If that complains about dependencies try pip install -r /path/to/local/robotframework-jsonlibrary/requirements.txt first.

Error message I was seeing:
```Collecting robotframework-JSONLibrary
  Downloading robotframework-jsonlibrary-0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-3y4ea4qg/robotframework-JSONLibrary/setup.py", line 5, in <module>
        from JSONLibrary.version import VERSION
      File "/tmp/pip-build-3y4ea4qg/robotframework-JSONLibrary/JSONLibrary/__init__.py", line 2, in <module>
        from JSONLibraryKeywords import JSONLibraryKeywords
    ImportError: No module named 'JSONLibraryKeywords'
    
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-3y4ea4qg/robotframework-JSONLibrary/
```

**Comment by 007bsd** [comment]
@MarkJB Thank you for the workaround. I run my robot tests inside docker container and use python based alpine image. It would be better if it is fixed with pip.

**Comment by totoroliu** [comment]
Is there any ETA when this fix (commit: e0c3852) will be released to pip?

**Comment by snehilmishra** [comment]
I am facing same issue 
do we have any workaround to resolve this issue

**Comment by nkontio** [comment]
Commit 0031c59b has been merged really recently which has python3 fixes. 
@nottyo could you create release for that merge so pip could be fixed for python3

**Comment by argoyanka** [comment]
Hi @nottyo,
When trying to install the package, I faced the following error. Your last commit didn't fix the issue. I had to use workaround suggested by @MarkJB to make it work.

Collecting robotframework-jsonlibrary
  Using cached https://files.pythonhosted.org/packages/e0/3c/609d90b967905c0b5c90f736666531e025756e71f3d538546cabff61ef3e/robotframework-jsonlibrary-0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pycharm-packaging/robotframework-jsonlibrary/setup.py", line 5, in <module>
        from JSONLibrary.version import VERSION
      File "/tmp/pycharm-packaging/robotframework-jsonlibrary/JSONLibrary/__init__.py", line 2, in <module>
        from JSONLibraryKeywords import JSONLibraryKeywords
    ImportError: No module named 'JSONLibraryKeywords'
    
    ----------------------------------------

Command "python setup.py egg_info" failed with error code 1 in /tmp/pycharm-packaging/robotframework-jsonlibrary/
You are using pip version 19.0.1, however version 19.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

**Comment by Srihari88** [comment]
Facing the same issue.

Collecting robotframework-jsonlibrary
  Using cached https://files.pythonhosted.org/packages/e0/3c/609d90b967905c0b5c90f736666531e025756e71f3d538546cabff61ef3e/robotframework-jsonlibrary-0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/1f/64bh46510zb0_lwr2b4b0nhr0000gn/T/pycharm-packaging/robotframework-jsonlibrary/setup.py", line 5, in <module>
        from JSONLibrary.version import VERSION
      File "/private/var/folders/1f/64bh46510zb0_lwr2b4b0nhr0000gn/T/pycharm-packaging/robotframework-jsonlibrary/JSONLibrary/__init__.py", line 2, in <module>
        from JSONLibraryKeywords import JSONLibraryKeywords
    ModuleNotFoundError: No module named 'JSONLibraryKeywords'
    
    ----------------------------------------

Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/1f/64bh46510zb0_lwr2b4b0nhr0000gn/T/pycharm-packaging/robotframework-jsonlibrary/

**Comment by ohmrefresh** [comment]
@Srihari88 This issue fixed in version 0.3.
